js/src/pages/Projects.tsx

```diff
@@ -0,0 +1,150 @@
+  closeProjectStatusForm,
+  createProjectAsync,
+  createProjectRoleAsync,
+  editProjectDescriptionAsync,
+  editProjectRoleAsync,
+  fetchProjectAsync,
+  openProjectCreateForm,
+  openProjectDescriptionForm,
+  openProjectRoleForm,
+  openProjectStatusForm,
+  removeProjectRoleAsync
+} from "../features/project/actions"
+
+import {
+  isOpenProjectCreateForm,
+  isOpenProjectRoleForm,
+  isOpenProjectStatusForm,
+  isLoading,
+  getErrors,
+  isOpenProjectDescriptionForm,
+  getProject
+} from "../features/project/selectors"
+import {
+  EditProjectDescriptionRequest,
+  CreateProjectRequest,
+  EditProjectStatusRequest,
+  EditProjectRoleRequest, RemoveProjectRoleRequest
+} from "ProjectModels"
+
+export const ProjectsPage = () => {
+  const dispatch = useDispatch()
+
+  const fetchProject = useCallback((slug: string) => {
+    dispatch(fetchProjectAsync.request(slug))
+  }, [dispatch])
+
+  const createProject = useCallback((form: CreateProjectRequest) => {
+    dispatch(createProjectAsync.request(form))
+  }, [dispatch])
+
+  const changeProjectStatus = useCallback((form: EditProjectStatusRequest) => {
+    dispatch(changeProjectStatusAsync.request(form))
+  }, [dispatch])
+
+  const editProjectDescription = useCallback((form: EditProjectDescriptionRequest) => {
+    dispatch(editProjectDescriptionAsync.request(form))
+  }, [dispatch])
+
+  const createProjectRole = useCallback((form: EditProjectRoleRequest) => {
+    dispatch(createProjectRoleAsync.request(form))
+  }, [dispatch])
+
+  const editProjectRole = useCallback((form: EditProjectRoleRequest) => {
+    dispatch(editProjectRoleAsync.request(form))
+  }, [dispatch])
+
+  const removeProjectRole = useCallback((form: RemoveProjectRoleRequest) => {
+    dispatch(removeProjectRoleAsync.request(form))
+  }, [dispatch])
+
+  const openCreateForm = useCallback(() => dispatch(openProjectCreateForm()), [dispatch])
+  const closeCreateForm = useCallback(() => dispatch(closeProjectCreateForm()), [dispatch])
+  const openRoleForm = useCallback(() => dispatch(openProjectRoleForm()), [dispatch])
+  const closeRoleForm = useCallback(() => dispatch(closeProjectRoleForm()), [dispatch])
+  const openStatusForm = useCallback(() => dispatch(openProjectStatusForm()), [dispatch])
+  const closeStatusForm = useCallback(() => dispatch(closeProjectStatusForm()), [dispatch])
+  const openDescriptionForm = useCallback(() => dispatch(openProjectDescriptionForm()), [dispatch])
+  const closeDescriptionForm = useCallback(() => dispatch(closeProjectDescriptionForm()), [dispatch])
+
+  const errors = useSelector(getErrors)
+  const loading = useSelector(isLoading)
+  const isOpenCreateForm = useSelector(isOpenProjectCreateForm)
+  const isOpenRoleForm = useSelector(isOpenProjectRoleForm)
+  const isOpenDescriptionForm = useSelector(isOpenProjectDescriptionForm)
+  const isOpenStatusForm = useSelector(isOpenProjectStatusForm)
+  const project = useSelector(getProject)
+
+  return (
+    <div>

```

Comment:
I don't understand why there are only Modals on the Projects page. I expected to see a list of projects here at least. If it is a work in progress you can display that.
[https://github.blog/2019-02-14-introducing-draft-pull-requests/](url)